src/teams.rs

```diff
@@ -74,6 +109,28 @@ impl Team {
 #[serde(transparent)]
 pub struct TeamLabel(pub String);
 
+pub fn start_updater_thread() {
+    // This is on a separate function that return a result to avoid panicking while having the
+    // teams write locked, which would poison the lock.
+    fn update() -> DashResult<()> {
+        let mut teams = SETUP.write().unwrap();
+        teams.update()?;
+        for (_name, team) in teams.teams() {
+            team.validate()?;
+        }
+        Ok(())
+    }
+    let _ = thread::spawn(|| {
+        let duration = Duration::from_secs(UPDATE_CONFIG_EVERY_MIN * 60);
+        loop {
```

Comment:
Hmm... `start_scraping` basically does the same script:
```rust
thread::spawn(|| {
    let duration = Duration::from_secs($interval * 60);
    loop {
        $logic
        thread::sleep(duration);
    }
})
```
extract idiom to function taking `$logic` + `$interval`?